distribution/docs/src/main/resources/content/_catalogFrameworks/catalog-framework-intro.adoc

```diff
@@ -31,7 +31,7 @@
 ....
 
 The `CatalogFramework` is the routing mechanism between catalog components that provides integration points for the Catalog Plugins.
-An <<_endpoints, endpoint>> invokes the active Catalog Framework, which calls any configured <<_pre_query_plugins, Pre-query>> or <<_pre_ingest_plugins, Pre-ingest plug-ins>>.
+An <<_endpoints, endpoint>> invokes the active Catalog Framework, which calls any configured <<_pre_query_plugins, Pre-query>> or <<{architecture-prefix}pre_ingest_plugins, Pre-ingest plug-ins>>.
 The selected <<_federation_strategy, federation strategy>> calls the active <<_catalog_providers, Catalog Provider>> and any connected or federated sources.
```

Comment:
❓ Should there be a prefix before these links?

Reply:
probably. fixed.